apps/speed.c

```diff
@@ -933,11 +1459,28 @@ int speed_main(int argc, char **argv)
         goto end;
     }
 
+    if (async_jobs > 0) {
+        if (!ASYNC_init(1, 0, 0)) {
```

Comment:
Should we initialise the pool with the number of jobs here? Or maybe that would be considered "cheating".